shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ObservabilitySetupService.java

```diff
@@ -0,0 +1,110 @@
+package com.redhat.service.smartevents.shard.operator;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.Objects;
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.shard.operator.providers.TemplateProvider;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.runtime.StartupEvent;
+
+/**
+ * Initialises Secret required by the Observability Operator in the Data Plane.
+ * Changes to the underlying {@link ConfigProperty} values are handled by the CPaaS
+ * AddOn deployment process and {@link SecretRestartHandler}.
+ */
+@ApplicationScoped
+public class ObservabilitySetupService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ObservabilitySetupService.class);
+
+    protected static final String OBSERVABILITY_ACCESS_TOKEN = "access_token";
+    protected static final String OBSERVABILITY_REPOSITORY = "repository";
+    protected static final String OBSERVABILITY_CHANNEL = "channel";
+    protected static final String OBSERVABILITY_TAG = "tag";
+    protected static final String OBSERVABILITY_NAME = "smartevents-observability-configuration";
+    protected static final String OBSERVABILITY_NAMESPACE = "smartevents-observability";
```

Comment:
@manstis I would suggest we move `OBSERVABILITY_NAME` and `OBSERVABILITY_NAMESPACE`  to `application.properties` as well and default them to these values. It keeps all the observability configuration in a single location.